docs/content/kv_commands.md

```diff
@@ -7,6 +7,30 @@ The `kv` subcommand allows you to store application data in the Cloudflare netwo
 * have a Wrangler project set up with your `account_id` configured in the `wrangler.toml`
 * call commands from within a Wrangler project directory.
 
+Most `kv` subcommands allow you to specify an `--env`, or environment, variable. This allows you to gate
+distinct namespaces within their own environments. For example, you could use seperate staging and production
+namespaces for KV data in your wrangler.toml:
+```toml
+type = "webpack"
+name = "my-worker"
+account_id = "<account id here>"
+route = "staging.example.com/*"
+workers_dot_dev = false
+
+kv-namespaces = [
+	{ binding = "STAGING_NAMESPACE", id = "06779da6940b431db6e566b4846d64db" }
```

Comment:
this is actually not a great example; your binding will be named the same in both environments given that the binding name is how you refer to the namespace in your worker, _not_ the title of the namespace.